R/controls.R

```diff
@@ -46,7 +42,8 @@ ionrangesliderLibs <- function() {
       script = "js/ion.rangeSlider.min.js",
       # ion.rangeSlider also needs normalize.css, which is already included in
       # Bootstrap.
-      stylesheet = c("css/ion.rangeSlider.css",
+      stylesheet = c(
+        "css/ion.rangeSlider.css",
```

Comment:
Just for git history sake

```suggestion
      stylesheet = c("css/ion.rangeSlider.css",
```